README.md

```diff
@@ -1,2 +1,216 @@
+    * gdd: cumulative growing degree days, using a base/threshold temperature of 10&deg;C 
+    * cum_precip: cumulative precipitation, mm
+#### B. Phenotypes
+* **tall_format_traits_season_6_timestamp.csv**
+    #### Columns
+    * date: year-month-day format
+    * sitename: location-based identifier, MAC Field Scanner Season 4 Range x Column y
+    * range: extracted from sitename
+    * column: extracted from sitename
+    * lat: latitude
+    * lon: longitude
+    * cultivar: sorghum variety
+    * trait: value
+        * canopy_height: cm
+        * aboveground_dry_biomass: kg/ha, kilogram per hectare, whole aboveground biomass at harvest 
+
+## Sharing and Access Information
+TERRA-REF Data Use [Policy](https://docs.terraref.org/user-manual/data-use-policy)
+
+## Methodological Information
+
+### I. Data Collected By: 
+### II. Data Collection Methods:
+* [Manual Field Data Protocols](https://docs.terraref.org/protocols/manual-field-data-protocols) from TERRA-REF docs
+
+### III. Data Curation: 
+
+* Trait data queried from [betydb](https://terraref.ncsa.illinois.edu/bety/) in `R` using this [tutorial](https://terraref.github.io/tutorials/accessing-trait-data-in-r.html)
+* Raw weather data downloaded from [MAC Weather Station](https://cals.arizona.edu/azmet/06.htm)
+* Geostreams weather data queried from [Clowder](https://pecanproject.github.io/modules/data.atmosphere/docs/reference/download.Geostreams.html) in `R` using this [tutorial](https://terraref.github.io/tutorials/accessing-weather-data-in-r.html)
+### IV. Data Transformation
+* Python code for data cleaning and transformation can be found within Jupyter notebooks [here] 
+* In-browser executable notebooks: [ ]
+
+## Data-specific Information
+
+Please see [Google Drive](https://drive.google.com/drive/folders/1B9Hmj2Xwu0J9arTfhFiFZqkEktSeag63?usp=sharing) for ontology annotations
+
+### Season 4
+
+#### Dates of Managements
+Please see metadata [notes](https://terraref.ncsa.illinois.edu/bety/api/v1/managements) for more details
+
+* **Tillage**: 2017-04-10, 2017-04-11
+* **Planting**: 2017-04-20
+* **Harvest**: 2017-09-11, 2017-09-12, 2017-09-15, 2017-09-16
+* **Plant thinning**: 2017-05-10, 2017-05-11, 2017-05-12
+* **Pesticide**: 2017-05-12
+* **Weed Control**
+    * Mechanical weed control: 2017-05-15
+    * Manual weed removal: 2017-05-18, 2017-05-19, 2017-05-26
+* **Fertilizer Treatments**
+    * Pre-planting: 2017-04-13
+    * Through drip irrigation: 2017-06-11, 2017-06-22, 2017-07-18
+* **Irrigation**
+    * First sprinkler irrigation after planting: 2017-04-21
+    * Final sprinkler irrigation to facilitate seedling emergence: 2017-05-04
+    * Final full-field: 2017-07-30
+    * Irrigation terminated across whole field: 2017-08-30
+    * Water-deficit experiments: 2017-08-01, 2017-08-15
+
+### Season 6
+#### Dates of Managements
+Please see metadata [notes](https://terraref.ncsa.illinois.edu/bety/api/v1/managements) for more details
```

Comment:
this should link to a specific season
also, use terraref.org/ as the base url because it is easier to read and will be persistent

Reply:
Actually, all of the key information from the experiment description here https://terraref.ncsa.illinois.edu/bety/experiments/6000000034 plus managements below should be included. It would be _nice_ if this were in an Rmd file that could be auto-generated by pulling from the database (nice = low priority now, but easier to reproduce for future seasons).

All expt and management data can be queried here: https://terraref.ncsa.illinois.edu/bety/api/v1/treatments?name=~Mac%20Season%206&associations_mode=full_info

but ... now might be a good time to start using the brapi endpoint where possible...

Reply:
here are examples from the brapi endpoint

https://terraref.org/brapi/v1/studies/6000000034

plot info and cultivar (germplasm) associations https://terraref.org/brapi/v1/studies/6000000034/layouts 

detailed germplasm info: https://brapi.workbench.terraref.org/brapi/v1/studies/6000000034/germplasm